### DISCUSSION TITLE
how to pass flags to chezmoi step?

### DISCUSSION THREAD

**Discussion by rubiin** [topgrade-rs/topgrade#831] [Q&A]
i wanted to pass --exclude=encrypted

**Comment by SteveLauC** [comment]
There is no way to pass a flag to the step, since they are sub-processes created by Topgrade, unless there is a configuration entry to allow users to do it, i.e., something similar to https://github.com/topgrade-rs/topgrade/blob/8b9d7ef8f37dbe8ef618000b974423d831c0dd7d/config.example.toml#L98

**Comment by GideonBear** [comment]
Replaced by #933
